tests/e2e/runner/chain.go

```diff
@@ -60,23 +57,20 @@ func (c *Chains) Register(name string, chain *ChainDetails) error {
 }
 
 // the Chain details are all hardcoded because they are currently fixed by kvtool.
-// some day they may be configurable, at which point `runner` can determine the ports
-// and generate these details dynamically
+// someday they may be accepted as configurable parameters.
 var (
-	kavaChain = ChainDetails{
-		RpcPort:  "26657",
-		RestPort: "1317",
```

Comment:
REST is essentially a GRPC wrapper with legacy endpoints removed, so makes sense to remove if we don't care about explicitly testing grpc gateway.  I only see a couple reasons why we'd test with it, but it's low priority.

Ethereum websocket (8546) may be added in future.  Not saying we include it here, just mentioning for completeness.

Reply:
yup, REST was unused and permanently removed here.

could stand to remove RPC as well, but it's currently setup to ping the `/status` endpoint.
i have a note to self to refactor it into a call to tendermint's node_info grpc.